e2e/testcafe-devextreme/tests/accessibility/loadPanel.ts

```diff
@@ -0,0 +1,28 @@
+import { Properties } from 'devextreme/ui/load_panel.d';
+import url from '../../helpers/getPageUrl';
+import { testAccessibility, Configuration } from '../../helpers/accessibility/test';
+import { Options } from '../../helpers/generateOptionMatrix';
+
+fixture.disablePageReloads`Accessibility`
+  .page(url(__dirname, '../container.html'));
+
+const options: Options<Properties> = {
+  showIndicator: [true, false],
```

Comment:
Should we declare visible option? By default visible option is false and loadPanel will not rendered after creating

Reply:
We should open the LoadPanel and other components after initialization. For this purpose, we may use the show() method or declare corresponding options during initialization to make the panel visible.